projects/canopy/src/lib/modal/modal-body-timer/modal-body-timer.component.html

```diff
@@ -1 +1 @@
-{{ timer }}
+{{ formattedTime }}
```

Comment:
I'm wondering whether a pipe would have been a better solution here

Reply:
@Devaney initially made a pipe in our project but I was the one that suggested to maybe just use the code directly into the component. I couldn't think of any other scenario where we would want to use this format so I thought this was the best place. We could always make it into a pipe if we need it in the future?

Reply:
It was a pipe to start with (hence the branch name) and moved it into the timer component later. The reasoning being, its rather specific formatting and couldn't imagine it being used elsewhere. If we find somewhere else to use it we could move it to a pipe at that point. What do you think?